examples/snippets/date-data-type/README.md

```diff
@@ -0,0 +1,19 @@
+# Sample - Date Data Type
+
+This example shows how to use the `DATE` data type with the Spanner ActiveRecord adapter. A `DATE` is a
+timezone-independent date. It does not designate a specific point in time, such as for example UTC midnight of the date.
```

Comment:
`such as for example UTC midnight` reads a bit redundant. Maybe change to `such as the UTC midnight`.